### PR TITLE
Support Translated Fields

### PR DESCRIPTION
**Pull request by atilafassina**
this PR addresses #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -104,15 +104,16 @@
         const keys = Object.keys(fields);
 
         keys.forEach(key => {
-          const value = fields[key].getValue();
-
-          if (typeof value === 'string') {
-            while ((match = URL_REGEXP.exec(value)) !== null) {
-              result.push(match[0])
+          const translations = Object.keys(fields[key]._fieldLocales)
```
Comment (stefanjudis) [inline index.html]: I missed that in when commenting the issue but we probably shouldn't go for `_locales` (underscore props usually mean "dont touch them") but rather with `entry.field.locales` (https://github.com/contentful/ui-extensions-sdk/blob/master/docs/ui-extensions-sdk-frontend.md#entryfieldsid-field).

And then use `getValue` with the locale.

**Review by stefanjudis — CHANGES_REQUESTED** [review]
@atilafassina Have a tiny change request. :)

I wonder if it's time to improve the table too then. Thinking of an entry with 4 fields  and 6 locales the table should maybe show which field in which locale includes an invalid url?

Thanks so much for handing this in. :)

File: index.html (hunk 1 above)
Comment (atilafassina) [reply index.html]: ouch, totally right.
I couldn't figure a way without using the private prop.

nice catch! 👌

**Comment by atilafassina** [comment]
Nice idea about the table improvements too.
I'll give it a go! 👍 

**EDIT**:
I'm afraid it's not possible to fetch the field title from the field, only the id, which we'll often differ from the title showing in the UI. E.g: a field entitled _"Link"_ may have an id `href`.

We could gather the results per field `id`, though I wonder how confusing can that be for non-developer users. I'm considering only add a new column with the translated language of the field, that would allow the editor to filter it making usage of the default translations filter, it should shorten enough the amount of fields to skim through looking for the bad URL.

Table would then look something like this:

Lang | URL | Code | Allowed
------|------|------|--------
en | https://foo.bar | 200 | yes
de | https://foo.de | 404 | yes

what do you think??

**Comment by stefanjudis** [comment]
Yeah works for me. :) 

As the space is quite limited I wonder though if the table was the right decision in the first place.  Maybe it should be a list to have more render options (but this might be out of scope of this PR – so let's go with the extra column. :) )

**Comment by atilafassina** [comment]
screen shot 2018-07-16 at 23 11 52

A bit tight. had to reduce the length of the URL cells by `50px`.
What do you think?

**Comment by stefanjudis** [comment]
Looks good to me. Thanks Atila!

**Pull request merged**